complex/ft_config.c

```diff
@@ -253,8 +253,17 @@ static int ft_parse_num(char *str, struct key_t *key, void *buf)
 		TEST_ENUM_SET_N_RETURN(str, FT_FUNC_SENDMSG, enum ft_class_function, buf);
 		TEST_ENUM_SET_N_RETURN(str, FT_FUNC_SENDV, enum ft_class_function, buf);
 		TEST_ENUM_SET_N_RETURN(str, FT_FUNC_SEND, enum ft_class_function, buf);
+		TEST_ENUM_SET_N_RETURN(str, FT_FUNC_INJECT_WRITEDATA, enum ft_class_function, buf);
+		TEST_ENUM_SET_N_RETURN(str, FT_FUNC_INJECT_WRITE, enum ft_class_function, buf);
```

Comment:
I believe it actually has to appear this way. This is what was causing it to run the wrong tests because it would match the string to the wrong enum. This list needs to be in descending order of string length.